### PR TITLE
docs(canary/signalfx): Add SignalFx to the metric group for canary and expect halyard/command/reference to pick up SignalFx changes in halyard.

### PR DESCRIPTION
**Pull request by mayitbeegh**

### CODE REVIEW

**Review by dorbin — COMMENTED** [review]
LGTM, pending review by
@duftler

**Review by duftler — APPROVED** [review]
LGTM.

**Pull request merged**
